app/code/Magento/Sales/etc/db_schema.xml

```diff
@@ -596,6 +596,9 @@
         <index referenceId="SALES_ORDER_ITEM_ORDER_ID" indexType="btree">
             <column name="order_id"/>
         </index>
+        <index referenceId="SALES_ORDER_ITEM_SKU" indexType="btree">
```

Comment:
@lbajsarowicz could you provide MySQL `EXPLAIN` for queries which use this table before and after adding the index for SKU column?

Reply:
Before changes:
![image](https://user-images.githubusercontent.com/1639941/71009095-fd751900-20e9-11ea-9545-6a92a5b0d346.png)

After changes:
![image](https://user-images.githubusercontent.com/1639941/71009289-43ca7800-20ea-11ea-985b-4eca4ecc51f2.png)

Adding index:
![image](https://user-images.githubusercontent.com/1639941/71009574-c2271a00-20ea-11ea-9ed4-5c94114f693c.png)

That is surprising for me, because actually - the change of column does not change anything. The index is also not necessary.

Reply:
As a result - it's fine to remove the index.